code/html/custom.js

```diff
@@ -2136,6 +2143,15 @@ function connectToURL(url) {
                 processData(data);
             }
         };
+		websock.onclose = function(evt) {
+			if(confirm("Connection lost with the device, click OK to refresh the page")) {
+				$("#layout").toggle(false);
+				location.reload();
+			}
+		}
+		websock.onopen = function(evt) {
+			ping_pong();
+		}
```

Comment:
Any reason this is not Interval?
```js
var ws_pingpong;
...
onclose() {
    clearInterval(ws_pingpong);
}
onopen() {
    ws_pingpong = setInterval(function() { sendAction("ping", {}); }, 5000);
}
```

Reply:
You are correct, setInterval is a better way to implement it, i will change it in the next commit